data/pages-content/cfp.md

```diff
@@ -0,0 +1,164 @@
+  Python groups, get involved - we want to help you make a difference.
+
+* If you’re from a background which is well-represented in most Python groups,
+  get involved - we want your help making a difference.
+
+* If you’re worried about not having enough technical knowledge, get involved - your fresh
+  perspective will be invaluable.
+
+* If you think you’re an imposter, join us - many of the EuroPython organisers
+  feel this way.
+
+* This is a volunteer led community, so join us - you will be welcomed with
+  friendship, support and compassion.
+
+You are welcome to share your questions and ideas with our programme team at
+[programme@europython.eu](mailto:programme@europython.eu).
+
+## The Conference Format ##
+
+The conference will feature around 16 tutorials/workshops and 120 talk sessions:
+
+* **Tutorials/Workshops (July 8th and 9th)**: hands-on **180 min sessions** with
+  40-100 people
+
+* **Talks (July 10th, until 12th)** : **30 or 45 min** : presentations of specific
+  and general interest to the European Python community (including a dedicated
+  PyData track)
+
+You are also welcome to submit **posters**, which will be printed in large formats. It is
+a graphical way to showcase your research, project or technology. Posters are
+exhibited at the conference, can be read at any time by participants, and can be
+discussed face-to-face with their authors during the poster sessions between
+July 10th and 12th.
+
+
+All presentations will be assigned one of the following levels:
+
+* **Beginner** - where the audience requires little or no previous knowledge to
+  engage with the subject matter.
+
+* **Intermediate** - some knowledge is assumed, although aspects of the talk
+  may introduce new (more advanced) topics.
+
+* **Advanced** - the audience is full of subject matter experts... go ahead and
+  really geek out.
+
+In previous years we’ve had many beginner and intermediate level talks. **We’d
+love to encourage more advanced level contributions too.** Come geek out with
+us at the conference!
+
+## Topics ##
+
+We welcome presentations on any topic that will be of specific or general
+interest to members of the Python community. We are a broad and deep community,
+so talks that may, at first, seem niche or unexpected actually make EuroPython
+the cosmopolitan place that it is. **It's your conference!**
+
+Browse through our [past
+conferences](https://www.europython-society.org/europython/) and [talk
+videos](https://www.youtube.com/c/EuroPythonConference) for more ideas and to
+learn where there may be gaps that ought to be filled.
+
+For ideas to contribute to the PyData track, take a look at PyData events:
+[pydata.org](https://pydata.org/).
+
+## Selection Process ##
+
+To ensure our programme reflects our diverse community, submissions go through
+three rounds of review and refinement before the final programme is announced:
+
+* **[Community voting](/voting)** (from March 8th, until Mach 15th): a voting round open to
```

Comment:
I'd say we better remove this link, because it's not online
```suggestion
* **Community voting** (from March 8th, until Mach 15th): a voting round open to
```

Reply:
It would probably be "March" instead of "Mach", though Mach sounds more exciting 😊

Reply:
It definitely does :D Thanks!